features/websocket/services/websocket.ts

```
@@ -0,0 +1,250 @@
+      this.socket.onopen = () => {
+        console.log('[WebSocket] Connected')
+        this.connectionState = 'connected'
+        this.reconnectAttempts = 0
+        this.reconnectDelay = 1000
+        this.lastPongTime = Date.now()
+
+        // Start ping interval
+        this.startPingInterval()
+
+        // Notify handlers
+        this.connectHandlers.forEach((handler) => handler())
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)